lib/nav_tab/link.rb

```diff
@@ -0,0 +1,30 @@
+class NavTab::Link
```

Comment:
Do these really belong under `lib`? It seems like they're somewhat of presenters.

I see the module NavTab is under lib, which is, I suspect, why you put it here. I would prefer if we starting moving application code out of `lib` and into `app`, be it `app/services` or `app/presenters`. 

On a similar note, what would be your thoughts on moving things out of `app/support` and into `app/services` or `app/xxx/concerns` (e.g. `app/support/products/relay_support` -> `app/models/concerns/products/relay_support`. Where it really bugs me is the `spec/app_support` directory feels really awkward (I confess, I think that was me that started that, and I now regret my decision).

Not something that needs to be done all at once, but something we do as we touch pieces of code.